### DISCUSSION TITLE
How Can I use multiple src video

### DISCUSSION THREAD

**Discussion by mahdiar** [vidstack/player#1256] [Q&A]
Hi
I use this code but quality selector doesn't show. Shoul I do somethnig else?
`    <source src="Doran-21-Cuba-Revolution-480.mp4" type="video/mp4"  data-width="853" data-height="480" />
    <source src="Doran-21-Cuba-Revolution-720.mp4" type="video/mp4"  data-width="1280" data-height="720" />
    <source src="Doran-21-Cuba-Revolution-1080.mp4" type="video/mp4"  data-width="1920" data-height="1080"  />
`

**Comment by mihar-22** [comment]
Works in our testing, please provide a [repro](<https://github.com/vidstack/player/blob/main/.github/REPRO.md>). Prefer help posts over in Discord if you can.
